electives/18843.md

```diff
@@ -0,0 +1,32 @@
+# 15-821/18-843: Mobile and Pervasive Computing
+
+| Category     | Difficulty |
+|:--           | :-:        |
+| Quizes       | 8          |
+| Presentation | 7          |
+| Project      | 8          |
+
+
+This course explores the principles and practice of mobile computing and its close relative, pervasive computing (aka “Internet of Things (IoT)”). The course will offer significant hands-on experience: in small teams, students will work under the guidance of a mentor on a project. Students will also, in small teams, present a summary and overview of the commercial landscape for one of the topics covered in class. There will be a brief quiz at the start of each class, based on the readings for that classes the principles and practice of mobile computing and its close relative, pervasive computing (aka “Internet of Things (IoT)”).
+
+## What to expect
+ - There are a total of 12 units. There will be an average of 6 papers to read for the following class. At the beginning of each class there will be a quiz based on the previous weeks readings
+ - You are expected to go to class and the quizzes can only be done in class and not remotely
+ - There is no textbook, all papers can be found online
+ - **There is no mid-term nor final exam**
+ - There is a semester long project in pairs, which is chosen the first day of classes 
+ - There is also a "Commercial Scan" presentation in which you research and present a relevant state of the art in industry, and how commercial practice diverges from academic research
+
+## Grading weights
+ - Project (55%)
+ - Commercial scan presentation (20%)
+ - Quizes (20%) -- you drop the worst test score
+ - Class participation (5%)
+
+## What to watch out for
+ - The quizes are clossed book so it's important to read them well to get the general idea but also take notes since the quizes tend to ask fine grained details
```

Comment:
```suggestion
 - The quizzes are closed-book so it's important to read them well to get the general idea but also take notes since the quizzes tend to ask fine grained details
```